docs/adguard-vpn-for-mac/solving-problems/logs.md

```diff
@@ -15,9 +15,9 @@ By default, AdGuard VPN for Mac uses the standard logging level, that is, the ba
 
 3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
 
-4. There is a check mark next to the **Attach technical logs**, which means that when you send a report, you also send logs.
+4. There is a check mark next to the **Send detailed system info**, which means that when you send a report, you also send logs.
 
->If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, select **Settings** → **General** → **Export logs** in the **Actions** section on the right.
+>If for some reason it is more convenient for you to send us logs in another way, you can export them yourself. To do this, go to **Support** → **Export logs and system info**.
```

Comment:
<img width="912" alt="image" src="https://user-images.githubusercontent.com/92315747/194080466-9d6912e8-4bb3-4460-9177-10e43fe357aa.png">
As you can see the current wording is correct